templates/upstream/upstream_footer.erb

```diff
@@ -1,10 +1,45 @@
-<% if @upstream_cfg_append %>
-<%# Slightly less obtuse way to sort but put keepalive at end -%>
-<%
-  @upstream_cfg_append = Hash[@upstream_cfg_append.sort]
-  @upstream_cfg_append['keepalive'] = @upstream_cfg_append.delete('keepalive')
--%>
-  <%- @upstream_cfg_append.each do |key,value| -%>
+<% if @hash -%>
```

Comment:
are you interested in turning this into an epp template? From our docs:

> * Is a new template added? The preferred language is [epp](https://puppet.com/docs/puppet/latest/lang_template_epp.html), not [erb](https://puppet.com/docs/puppet/latest/lang_template_erb.html)

it isn't required to turn this into an epp template, but it would be nice to have.

Reply:
If I convert upstream_member.erb I will convert this too. It makes no sense to use different templatetypes for one file I think.